app/assets/javascripts/application.js

```diff
@@ -31,6 +21,8 @@ jQuery(function ($) {
 
   var $atomAutodiscoveryLink = $("link[type='application/atom+xml']").eq('0')
 
+  var $transitionSubscribeButton = document.querySelector('#checklist-email-signup button.js-subscribe-button')
```

Comment:
Could we use a `js-` class as a hook for the JavaScript instead? This can be [passed into the button component](https://components.publishing.service.gov.uk/component-guide/button/with_js_classes) in the `class` parameter.

For example:
```suggestion
  var $transitionSubscribeButton = document.querySelector(".js-subscribe-button-tracker")
```